sys/include/net/gnrc/mac/mac.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_mac  A MAC module for providing common MAC parameters and helper functions
+ * @ingroup     net
+ * @brief       A MAC module for providing common MAC parameters and helper functions.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Internal definitions of GNRC_MAC
```

Comment:
AFAIK as I can see these are not only internal but can (and shall) also be set externally.